packages/dds/tree/src/simple-tree/api/schemaFactory.ts

```diff
@@ -103,6 +103,8 @@ export function schemaFromValue(value: TreeValue): TreeNodeSchema {
 
 /**
  * Options when declaring an {@link SchemaFactory.object|object node}'s schema
+ *
+ * @alpha
```

Comment:
Nit: Question about the documentation on the property below. The first line reads:
> Opt in to viewing documents which have unknown optional fields for this object's identifier, i.e. optional fields
> in the document schema which are not present in this object schema declaration.

I'm not sure what "identifier" means in this context. Should this instead just say something like:
> Allow nodes typed with this object node schema to contain optional fields that are not present in the schema declaration.

?

Reply:
I like that way of putting it. `identifier` refers to the schema's identifier, but it's not particularly necessary to mention that here.